compiler/ml/pprintast.ml

```diff
@@ -543,7 +543,7 @@ and expression ctxt f x =
     | Pexp_let _ | Pexp_letmodule _ | Pexp_open _ | Pexp_letexception _
         when ctxt.semi ->
         paren true (expression reset_ctxt) f x
-    | Pexp_fun (l, e0, p, e, arity) ->
+    | Pexp_fun {arg_label=l; default= e0; lhs= p; rhs= e; arity} ->
```

Comment:
Is this not formatted with ocamlformat? As there are different spaces around `=`.

Reply:
Good question.
Perhaps it gets stuck somewhere in the file.

Reply:
It was not formatted as there are some cppo ifdef's.

There are all these cppo configs:
```
(env
 (dev
  (env-vars
   (CPPO_FLAGS -U=RELEASE)))
 (release
  (env-vars
   (CPPO_FLAGS -D=RELEASE))
  (ocamlopt_flags
   (:standard -O3 -unbox-closures)))
 (static
  (env-vars
   (CPPO_FLAGS -D=RELEASE))
  (ocamlopt_flags
   (:standard -O3 -unbox-closures)))
 (browser
  (env-vars
   (CPPO_FLAGS -D=BROWSER))
  (ocamlopt_flags
   (:standard -O3 -unbox-closures))))
```
do we actually need them? All of them?